squeaknode/admin/webapp/app.py

```diff
@@ -118,6 +119,14 @@ def hello_world():
         logger.info("Getting hello route.")
         return "Hello, World!"
 
+    @app.route("/store_default_signing_profile_id/<profile_id>/profile_id", methods=["POST"])
+    @login_required
+    def store_default_signing_profile_id(profile_id):
+        logger.info(f"Storing default signing profile: {profile_id}")
+        resp = make_response(render_template(...))
```

Comment:
You shouldn't need any template. Maybe use this: https://flask.palletsprojects.com/en/2.0.x/api/#flask.after_this_request